### PR TITLE
attempt to fix  env variables with windows

### PR DESCRIPTION
**Pull request by SeanDoyleGit**
Attempt to fix setting environment variables on windows

### CODE REVIEW

File: packages/create-keystone-app/templates/todo/package.json.ejs [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "<%= appName%>",
   "version": "0.0.0",
   "scripts": {
-    "start": "NODE_ENV=development DISABLE_LOGGING=true keystone"
+    "start": "<%= envVariables%> keystone"
```
Comment (jesstelford) [inline packages/create-keystone-app/templates/todo/package.json.ejs]: Perhaps we can use [`cross-env`](https://www.npmjs.com/package/cross-env), which has the upshot of making the code more portable.

```suggestion
    "start": "cross-env NODE_ENV=development DISABLE_LOGGING=true keystone"
```

Comment (SeanDoyleGit) [reply packages/create-keystone-app/templates/todo/package.json.ejs]: will change now :)

File: packages/create-keystone-app/lib/generator.js [hunk 2]
```diff
@@ -61,6 +61,7 @@ function generate(name, noDeps) {
         copyTemplate(`${templateDir}/todo`, projectDir, {
           name,
           appName,
+          envVariables,
```
Comment (mitchellhamilton) [inline packages/create-keystone-app/lib/generator.js]: Is this meant to be here?

File: packages/create-keystone-app/templates/todo/package.json.ejs [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "name": "<%= appName%>",
   "version": "0.0.0",
   "scripts": {
-    "start": "NODE_ENV=development DISABLE_LOGGING=true keystone"
+    "start": "cross-env NODE_ENV=development DISABLE_LOGGING=true keystone"
   },
   "private": true,
   "dependencies": {
```
Comment (mitchellhamilton) [inline packages/create-keystone-app/templates/todo/package.json.ejs]: `cross-env` needs to be in the dependencies here, not in create-keystone-app's dependencies.

File: packages/create-keystone-app/lib/generator.js (hunk 2 above)
Comment (SeanDoyleGit) [reply packages/create-keystone-app/lib/generator.js]: no it is not

**Comment by jesstelford** [comment]
@SeanDoyleGit Once you've added the `cross-env` dep to the `packages/create-keystone-app/templates/todo/package.json.ejs` project, it'd be great to see it also used in each of the projects in `demo-projects` & `test-projects` 🎉

File: packages/create-keystone-app/package.json [hunk 4]
```diff
@@ -17,6 +17,7 @@
     "@samverschueren/stream-to-observable": "^0.3.0",
     "arg": "^4.1.0",
     "chalk": "^2.4.2",
+    "cross-env": "^5.2.0",
```
Comment (jesstelford) [inline packages/create-keystone-app/package.json]:
```suggestion
```

This isn't needed here (it's meant to be in `packages/create-keystone-app/templates/todo/package.json.ejs`)

**Pull request merged**